src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -12,7 +12,41 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal static class Extensions
     {
+        public static string CLREventProviderName = "Microsoft-Windows-DotNETRuntime";
+
         private static EventLevel defaultEventLevel = EventLevel.Verbose;
+        // Keep this in sync with runtime repo's clretwall.man
+        private static Dictionary<string, long> CLREventKeywords = new Dictionary<string, long>()
+        {
+            { "gc", 0x1 },
```

Comment:
nit: Make the keywords camel casing, and add a key comparison case insensitive.

Reply:
**Those keywords are being used on the console UI, and it will make them more readable.